src/components/FlowLoader.tsx

```diff
@@ -51,36 +45,38 @@ export class FlowLoader extends React.PureComponent<FlowLoaderProps, FlowLoaderS
             this.setDefinition.bind(this),
             this.save.bind(this),
             this.props,
-            QUIET_UI, QUIET_SAVE
+            QUIET_UI,
+            QUIET_SAVE
         );
         this.setDefinition(definition);
     }
 
     componentDidMount() {
-        Config.get().external.getFlow(this.props.uuid, false).then((details: FlowDetails) => {
-            this.initialize(details.definition);
-        });
+        Config.get()
+            .external.getFlow(this.props.uuid, false)
+            .then((details: FlowDetails) => this.initialize(details.definition));
     }
 
     private setDefinition(definition: FlowDefinition, dependencies?: FlowDefinition[]) {
         // TODO: determine full dependency list and fetch those at simulation time
         if (!dependencies) {
-            this.setState({ definition: definition });
+            this.setState({ definition });
         } else {
-            this.setState({ definition: definition, dependencies: dependencies });
+            this.setState({ definition, dependencies });
         }
     }
 
     render() {
-        var flow = null;
-        if (this.state.definition) {
-            flow = <Flow
-                definition={this.state.definition}
-                dependencies={this.state.dependencies}
-                mutator={this.mutator} />
-        }
-        return flow;
+        return (
+            this.state.definition ? (
```

Comment:
Is this a code formatter forcing a ternary here?

Reply:
yes, but happy to remove